templates/apiTypes.hbs

```diff
@@ -6,7 +6,7 @@ import { Api } from '../../models'
 export interface {{className name}}Api {
 	{{#each operations}}
 	{{identifier name}}: ({{#each parameters}}{{identifier name}}: {{{nativeType}}}{{#unless required}} | undefined{{/unless}}{{#hasMore}}, {{/hasMore}}{{/each}}{{!--
-	--}}{{#if requestBody.nativeType}}{{#if parameters}}, {{/if}}{{identifier requestBody.name}}: {{{requestBody.nativeType}}}{{#unless requestBody.required}} | undefined{{/unless}}{{/if}}{{!--
+	--}}{{#if requestBody.nativeType}}{{#if parameters}}, {{/if}}{{identifier requestBody.name}}: {{{requestBody.nativeType}}}{{#unless (isRequestRequired .)}} | undefined{{/unless}}{{/if}}{{!--
```

Comment:
Firstly this was a dumb way for me to make us populate a list of parameters... having to check if we had a previous one to add a `,`... let's change this to use the `{{#join ...}` approach. I've used that in some core generators and I hope it will improve this.